### DISCUSSION TITLE
interactivebrokers.com takes credit for ib_insync and misrepresents ib_async

### DISCUSSION THREAD

**Discussion by glenntam** [ib-api-reloaded/ib_async#96] [General]
interactivebrokers.com takes credit for ib_insync and misrepresents ib_async:  [https://www.interactivebrokers.com/campus/ibkr-quant-news/ib_insync-guide-interactive-brokers-api/](https://www.interactivebrokers.com/campus/ibkr-quant-news/ib_insync-guide-interactive-brokers-api/) 

"ib_insync is a framework that simplifies the Interactive Brokers (IB) Native Python API that was _developed by the IB team_."

"IB Insync has been discontinued for a few months now due to personal issues of the library’s developer." Calling Ewald's death a "personal issue" really spits in the face of the community that loved him.

"...the co-developer has created a split “IB Async” library to continue development work and operates on modern releases of the TWS API library to..." Calling Matt a "co-developer" of ib_insync makes it sound like he was on the core team of ib_insync and that he and Ewald were a duo that owned ib_insync. I doubt Matt would say it like that.

It's appalling that IB would use such irresponsible reporting from a third-party content author and have it appear on the official interactivebrokers.com site. If the author had spent 5 minutes to google instead of chatgpt they could have gotten their facts right. I wish there was some petition our community could start to rectify this garbage post. I don't suppose anybody has any contacts at IB management that could set this straight?

**Comment by mattsta** [comment]
Thanks for tracking that down!

Amusingly, even in the middle of that page it does also say: "Note that ib_insync was not created by the IB team."

The "IBKR campus" site is always a bit weird, but it looks like this post was either partially copied, borrowed, or inspired by https://algotrading101.com/learn/ib_insync-interactive-brokers-api-guide/

Plus, the difference between "simple getting started" tutorials and properly using all the async functions in well-structured encapsulation for live production code is a larger step than any quick guide can provide (the first step is "spend six months learning how async programming works, then actually start working"). Real usage ends up being split between "simple hacks" people try on their own versus better constructed "large systems" being built in secret that probably don't get shared.

Also I think IBKR ($70 billion market cap) always tires to distribute much of their work globally as possible (cheaper?), probably creating difficult communication/understanding dynamics across timezones and languages: https://job-boards.greenhouse.io/ibkr?departments%5B%5D=4027657002

**Reply by sparkie3** [reply]
This really does look like a miscommunication to me. It could certainly use some different prose, but I don't think this is anything more than someone whose native language is not English.
